Developer Samples/AddCollectionMembers/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="InRule.Common" version="5.4.0" targetFramework="net461" />
-  <package id="InRule.Repository" version="5.4.0" targetFramework="net461" />
-  <package id="InRule.Runtime" version="5.4.0" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net461" />
-  <package id="Newtonsoft.Json" version="12.0.1" targetFramework="net461" />
+  <package id="InRule.Common" version="5.4.0" targetFramework="net472" />
```

Comment:
these and the other samples aren't pointing at the latest version of InRule. 
Could that create an issue when package dependency resolution happens where it might still target (via deps) net45/net461 asms?